ladybug/stat.py

```diff
@@ -20,11 +32,35 @@ class Stat(object):
         monthly_tau_diffuse
     """
 
-    def __init__(self, file_path=None):
+    def __init__(self, file_path):
         """Initalize the class."""
         self.file_path = file_path
-        self._is_data_loaded = False
-        self._header = None  # epw header
+
+        # defaults in case some climate parameters are unclassifiable
+        self._header = None
+        self._location = None
+        self._stand_press_at_elev = None
+        self._ashrae_climate_zone = None
+        self._koppen_climate_zone = None
+        self._winter_des_day_dict = None
+        self._summer_des_day_dict = None
+        self._monthly_db_04 = []
+        self._monthly_wb_04 = []
+        self._monthly_db_20 = []
+        self._monthly_wb_20 = []
+        self._monthly_db_50 = []
+        self._monthly_wb_50 = []
+        self._monthly_db_100 = []
+        self._monthly_wb_100 = []
+        self._monthly_db_range_50 = []
+        self._monthly_wb_range_50 = []
+        self._monthly_wind = []
+        self._monthly_wind_dirs = []
+        self._monthly_tau_beam = []
+        self._monthly_tau_diffuse = []
+
+        # import the data from the file
+        self.import_data()
```

Comment:
You most likely want to make `import_data` internal e.g. `_import_data`. If someone calls this method on an instance you will end up with duplicate input data.

Reply:
Done!